### PR TITLE
feature(va04hl): Improved Rx Time tracking

### PR DESCRIPTION
**Pull request by ksukumar0**

### CODE REVIEW

File: src/radio/sx126x/radio.c [hunk 1]
```diff
@@ -1289,7 +1290,7 @@ void RadioIrqProcess( void )
             // This is the Tx Transmit Time
             hm_srt_operational_data_report_accumulate_radio_msg_stats_in_cache(TxBytes, false);
             hm_srt_operational_data_report_accumulate_radio_air_times_in_cache(k_uptime_get() - TxKernelTimer, false);
-
+            set_gpio(false);
```
Comment (lukasheim) [inline src/radio/sx126x/radio.c]: [Request]: Please remove the `set_gpio()`.

File: src/radio/sx126x/radio.c [hunk 2]
```diff
@@ -1331,11 +1333,12 @@ void RadioIrqProcess( void )
                 }
                 SX126xGetPayload( RadioRxPayload, &size , 255 );
                 SX126xGetPacketStatus( &RadioPktStatus );
-                hm_srt_operational_data_report_accumulate_radio_air_times_in_cache(k_uptime_get() - RxKernelTimer, true);
-                hm_srt_operational_data_report_accumulate_radio_msg_stats_in_cache(size, true);
                 if( ( RadioEvents != NULL ) && ( RadioEvents->RxDone != NULL ) )
                 {
                     RadioEvents->RxDone( RadioRxPayload, size, RadioPktStatus.Params.LoRa.RssiPkt, RadioPktStatus.Params.LoRa.SnrPkt );
+                    hm_srt_operational_data_report_accumulate_radio_air_times_in_cache(k_uptime_get() - RxKernelTimer, true);
```
Comment (lukasheim) [inline src/radio/sx126x/radio.c:1337]: [Suggestion]: would be nice to have this via a callback. Not a fix for now, but keep it in mind for future changes.

File: src/radio/sx126x/radio.c [hunk 3]
```diff
@@ -1047,14 +1049,14 @@ void RadioSend( uint8_t *buffer, uint8_t size )
     else
     {
         SX126x.PacketParams.Params.Gfsk.PayloadLength = size;
-    }
+    }
```
Comment (lukasheim) [inline src/radio/sx126x/radio.c]: [Optional]: Remove the space that sneaked in.

File: src/radio/sx126x/radio.c (hunk 1 above)
Comment (ksukumar0) [reply src/radio/sx126x/radio.c]: Will remove this, but it's an empty function in the `set_gpio` function in the `hm-srt-operational-data-report.c`

**Review by lukasheim — APPROVED** [review]

**Pull request merged**